Startup/FullTime/FullTimeCheck.js

```diff
@@ -1,121 +1,151 @@
-                    sendVirtualAssistantMessage("Transferring gold...");
-                    addGold(randomInteger(50, 200));
-                }
-
-                setVar(VARIABLE_WEEKLY_CHORES_TIME, 0);
-                setVar(VARIABLE_WEEKLY_CHORES_DONE, 0);
-
-                //TODO: Study and athlete mode
-            }
-        }
-
-        if (isVar(VARIABLE_FULL_TIME_TRIAL_UNTIL) && getDate(VARIABLE_FULL_TIME_TRIAL_UNTIL).hasPassed()) {
-            delVar(VARIABLE_FULL_TIME_TRIAL_UNTIL);
-            sendVirtualAssistantMessage("Slave");
-            sendVirtualAssistantMessage("Your trial period has expired");
-            sendVirtualAssistantMessage("Meaning you now have a choice");
-            sendVirtualAssistantMessage("Stay full time or go back to part time");
-            sendVirtualAssistantMessage("Which is it?", false);
-            const answer = createInput();
-
-            while (true) {
-                if (answer.containsIgnoreCase("part")) {
-                    sendVirtualAssistantMessage("Back to part time then...");
-                    setPartTime();
-                    break;
-                } else if (answer.containsIgnoreCase("full")) {
-                    sendVirtualAssistantMessage("Excellent choice!");
-                    break;
-                } else {
-                    sendVirtualAssistantMessage("Part time or full time?");
-                    answer.loop();
-                }
-            }
-        }
-    }
-}
-
-function slaveIsBack(tooLate, vacation) {
-    if(tooLate) {
-        sendVirtualAssistantMessage(random("You're late!", "You're late %SlaveName","You haven't returned in due time", "I expected you sooner!"));
-        sendVirtualAssistantMessage(random("I don't allow for you to be gone longer than permitted!", "You aren't allowed to be gone without proper agreement", "You know the rules!"));
-        sendVirtualAssistantMessage(random("I'm giving you punishment points", "I'm assigning you punishment points", "I have to give you punishment points"));
-        setVar(VARIABLE_PUNISHMENT_POINTS, getVar(VARIABLE_PUNISHMENT_POINTS) + 150);
-    } else {
-        if(vacation) {
-            sendVirtualAssistantMessage(random("You're back!", "Welcome back", "Welcome back %SlaveName%"));
-            sendVirtualAssistantMessage(random("Good to see you again", "It's great to see you again", "I'm happy to see you back"));
-        } else {
-            sendVirtualAssistantMessage(random("You're back from your vacation", "You're back from your holiday", "You're back!"));
-            sendVirtualAssistantMessage(random("I hope you enjoyed it because it's right back to service now!", "I thoroughly hope you enjoyed it and are ready to return to service"));
-        }
-    }
+//sendMessage("in full time checkup");
+       const date = new Date();
+//sendMessage("the day is "+ date.getDay());
+
+    if (isVar(VARIABLE_LAST_ROUTINE_CHECK) && getDate(VARIABLE_LAST_ROUTINE_CHECK).hasPassed()) {

```

Comment:
What was wrong with the original if statement?

Reply:
honestly it's been so long I don't remember... I suspect without this change the custom first time sequence at the bottom will never trigger....